app/(tabs)/summary/main-summary.tsx

```
@@ -0,0 +1,158 @@
+import {useState} from 'react';
+import {useLocalSearchParams} from 'expo-router';
+import {ScrollView, View} from 'react-native';
+import {Button, Text} from 'react-native-paper';
+
+import {BarChart, Chip, DatePicker} from '@/components';
+import {lastDayOfMonth} from 'date-fns';
+import {useAppSelector} from '@/hooks';
+import {aggregateExpenses} from '@/redux/main/selectors';
+import {EXCLUDED_CAT, formatPrice} from '@/common';
+import {parseInt} from 'lodash';
```

Biome

[error] Do not shadow the global "parseInt" property.

Consider renaming this variable. It's easy to confuse the origin of variables when they're named after a known global.

(lint/suspicious/noShadowRestrictedNames)